src/main/java/seedu/address/model/person/Nric.java

```diff
@@ -4,36 +4,37 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Represents a Person's nric in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidNric(String)} (String)}
  */
-public class Address {
+public class Nric {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Nric must be in the form [S/T/F/G][7-numbers][A-Z], "
+            + "and it should not be blank";
 
     /*
```

Comment:
Perhaps a better comment would explain the regex on what makes a valid NRIC, such as:
1. It must start with S, T, F or G
2. It must have 7 digits afterward
3. it must end with a letter